src/plugins/kibana/public/settings/sections/indices/add_data_steps/upload_data_step/upload_data_step.js

```diff
@@ -0,0 +1,44 @@
+import modules from 'ui/modules';
+import template from './upload_data_step.html';
+import _ from 'lodash';
+import IngestProvider from 'ui/ingest';
+
+modules.get('apps/settings')
+  .directive('uploadDataStep', function () {
+    return {
+      template: template,
+      scope: {
+        results: '='
+      },
+      bindToController: true,
+      controllerAs: 'uploadStep',
+      controller: function ($scope, $http, Notifier, $window, Private) {
+        const ingest = Private(IngestProvider);
+        const notify = new Notifier({
+          location: 'Add Data'
+        });
+
+        const usePipeline = !_.isEmpty(_.get(this.results, 'pipeline.processors'));
```

Comment:
Yeah I'd like to leave it, it'll correctly return false for now and I'd only have to rewrite it when pipelines are re-added.